versioned_docs/version-8.7/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md

```diff
@@ -22,76 +19,14 @@ camunda_operate_123_8.1.0_part_6_of_6
 
 Operate provides an API to perform a backup and manage backups (list, check state, delete). Restore a backup using the standard Elasticsearch API.
 
+<!-- TODO: This note is shit and factual wrong; application HTTP port is 80 or 8080?! -->
```

Comment:
This note actually makes sense. Port 9600 is used as management port and therefore, the backup api should also be on this port.

Reply:
My problem was around this one `The backup API can be reached via the Actuator management port, which by default is the same as application HTTP port`.
Actuator port: 9600
Application HTTP port: 8080 / 80

Reply:
Adjusted it as for me it's misleading to what reality is.